docs/core/deploying/index.md

```diff
@@ -1,95 +1,120 @@
+    Commands that produce an executable:
 
-## Framework-dependent deployments (FDD)
+    | Type                                    | SDK Version   | Command |
+    | --------------------------------------- | ------------- | ------- |
+    | [runtime-dependent](#runtime-dependent) | 3.1, 3.0      | [`dotnet publish`](../tools/dotnet-publish.md) |
+    |                                         | 2.1           | Not supported. |
+    | [self-contained](#self-contained)       | 3.1, 3.0, 2.1 | [`dotnet publish -r <RID>`](../tools/dotnet-publish.md) |
 
-For an FDD, you deploy only your app and third-party dependencies. Your app will use the version of .NET Core that's present on the target system. This is the default deployment model for .NET Core and ASP.NET Core apps that target .NET Core.
+    In general, platform-specific apps can be created for platforms other than your current platform. The `-r <RID>` switch sets the target [runtime-identifier](../rid-catalog.md).
 
-### Why create a framework-dependent deployment?
+01. A runtime-dependent cross-platform *dll* file.
 
-Deploying an FDD has a number of advantages:
+    Commands that produce a runtime-dependent *dll* app:
 
-- You don't have to define the target operating systems that your .NET Core app will run on in advance. Because .NET Core uses a common PE file format for executables and libraries regardless of operating system, .NET Core can execute your app regardless of the underlying operating system. For more information on the PE file format, see [.NET Assembly File Format](../../standard/assembly/file-format.md).
+    | Type                                    | SDK Version    | Command |
+    | --------------------------------------- | -------------- | ------- |
+    | [runtime-dependent](#runtime-dependent) | 3.1, 3.0, 2.1  | [`dotnet publish`](../tools/dotnet-publish.md) |
 
-- The size of your deployment package is small. You only deploy your app and its dependencies, not .NET Core itself.
+    Cross-platform *dll* apps are run with the `dotnet file.dll` command.
 
-- Unless overridden, FDDs will use the latest serviced runtime installed on the target system. This allows your application to use the latest patched version of the .NET Core runtime. 
+## Self-contained
 
-- Multiple apps use the same .NET Core installation, which reduces both disk space and memory usage on host systems.
+Self-contained publishing always produces a platform-specific executable. The output publishing folder contains all components of the app, including the .NET Core libraries and target runtime. The app is isolated from other .NET Core apps and doesn't use a locally installed shared runtime. The user of your app isn't required to download and install .NET Core.
```

Comment:
"publishing" isn't really the thing that is self-contained.  Perhaps "Publishing an app as a self-contained executable always..." Or "When you publish an app as a self-contained executable, the executable is always runtime-specific."